### PR TITLE
Added apple social button

### PR DESCRIPTION
**Pull request by mariotaddeucci**

### CODE REVIEW

File: bin/bulma-social.css [hunk 1]
```diff
@@ -71,6 +71,72 @@ fieldset[disabled] .is-adn.focus {
   background-color: #fff;
 }
 
+
+.is-apple {
+  color: #fff;
+  background-color: #444;
```
Comment (aldi) [inline bin/bulma-social.css]: Change this line to ``` background-color: #000 ```

**Review by aldi — CHANGES_REQUESTED** [review]
Hey! 👋
Thanks for the pull request.
Based on [Apple's guidelines](https://developer.apple.com/design/human-interface-guidelines/sign-in-with-apple/overview/buttons/), let's make ``` background-color: #000```.
Can you please add `is-apple` to the main page too?

File: bin/bulma-social.css (hunk 1 above)
Comment (mariotaddeucci) [reply bin/bulma-social.css]: Hi Aldi, I fixed the colors to the correct and added the the is-apple class to the main page.

**Comment by aldi** [comment]
Awesome, thanks!

**Pull request merged**